microsoftgraph/microsoft-graph-explorer-v4
# Task: Add copy telemetry

## Overview
Closes #2411 

### Demo
![image](https://user-images.githubusercontent.com/45680252/220072404-cb220409-91b6-4734-857a-4e6fa8429ea4.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions
Use the keyboard to copy any content on Access tokens, Response preview, Response headers, Code snippets, Adaptive cards and Response body tabs.
Open the network tab and inspect the track call
Notice that the keyboard event is available with the property `trgger` to differentiate it from other copy events
